### ISSUE TITLE
Errors after ./configure, AlmaLinux 9.1, kernel 5.14.0-162.12.1.el9_1.x86_64

### ISSUE THREAD

**Issue by serjponomarev** [Open-CAS/open-cas-linux#1442]
Errors after ./configure
AlmaLinux 9.1
kernel 5.14.0-162.12.1.el9_1.x86_64

[OpenCAS  22.06.2.0723](https://github.com/Open-CAS/open-cas-linux/releases/tag/v22.6.2)
```
./configure
Preparing configuration
ERROR! Following steps failed while preparing config:
1_bdev_nr_sectors.conf
1_bd_part_count.conf
1_bdev_whole.conf
1_bd_first_part.conf
1_bio_dev.conf
1_bio_clone.conf
2_bio_barrier.conf
```
Logs in attach
[logs.zip](https://github.com/Open-CAS/open-cas-linux/files/10693246/logs.zip)

**Comment by aneagoe** [comment]
Several issues around this. Firstly, it looks like `linux/genhd.h` has been removed from the kernel (https://lore.kernel.org/all/9bb3f4c6-4e21-e1d7-b4e1-f65d273cd3f8@nvidia.com/t/) but opencas is still looking for it.
Then, there are failures around bio, which seem related to some removals:
* https://patchwork.kernel.org/project/linux-block/patch/149776073934.23258.17954806456328395509.stgit@noble/
* https://patchwork.kernel.org/project/linux-block/patch/20180724075234.26861-4-hch@lst.de/
Either way, it looks like there's some work required to get this working on CentOS9 and its derivatives.

**Comment by katlapinka** [comment]
This issue is fixed on current master branch (6c0829d5eae151f07f0b8a4825ae106980764c28) and it is planned to be released in next official release 24.9
